runtime/server/metrics_views_test.go

```diff
@@ -0,0 +1,322 @@
+package server
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"github.com/rilldata/rill/runtime/drivers"
+)
+
+func createServerWithMetricsView(t *testing.T) (*Server, string) {
+	metastore, err := drivers.Open("sqlite", "file:rill?mode=memory&cache=shared")
+	require.NoError(t, err)
+
+	err = metastore.Migrate(context.Background())
+	require.NoError(t, err)
+
+	server, err := NewServer(&ServerOptions{
+		ConnectionCacheSize: 100,
+	}, metastore, nil)
+	require.NoError(t, err)
+
+	resp, err := server.CreateInstance(context.Background(), &runtimev1.CreateInstanceRequest{
+		OlapDriver:   "duckdb",
+		OlapDsn:      "",
+		RepoDriver:   "file",
+		RepoDsn:      "../testproject/ad_bids",
```

Comment:
Name the test data folder `testdata` instead of `testproject` – it's a convention in go (e.g. see [here](https://dave.cheney.net/2016/05/10/test-fixtures-in-go))

Reply:
fixed